be/src/olap/rowset/segment_v2/binary_dict_page.cpp

```diff
@@ -313,5 +315,23 @@ Status BinaryDictPageDecoder::next_batch(size_t* n, ColumnBlockView* dst) {
     return Status::OK();
 }
 
+BinaryDictPageDecoder* BinaryDictPageDecoder::clone_for_cache() const {
```

Comment:
doris's code style is  Status BinaryDictPageDecoder::clone_for_cache(std::unique_ptr< BinaryDictPageDecoder >& BinaryDictPageDecoder)